test/table.ts

```diff
@@ -1120,7 +1120,28 @@ describe('Bigtable/Table', () => {
           .on('data', done);
       });
     });
-
+    it('Should respect the timeout parameter passed in', done => {
+      // The timeout is 2 seconds, but the error is received after 3 seconds
+      // so the client doesn't retry because more than 2 seconds have elapsed.
+      const requestSpy = (table.bigtable.request = sinon.spy(() => {
+        const stream = new PassThrough({
+          objectMode: true,
+        });
+        (stream as any).abort = () => {};
+        setTimeout(() => {
+          const error = new Error('retry me!') as ServiceError;
+          error.code = 4;
+          stream.emit('error', error);
+        }, 3000);
+        return stream;
+      }));
+      const stream = table.createReadStream({gaxOptions: {timeout: 2000}});
+      stream.on('error', (error: ServiceError) => {
+        assert.strictEqual(error.code, 4);
```

Comment:
```suggestion
        assert.strictEqual(error.code, 4);
        assert.notStrictEqual(error.message, "retry me!");
```

just another check for posterity